src/org/usfirst/frc/team2473/robot/commands/PointTurn.java

```diff
@@ -0,0 +1,89 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team2473.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import org.usfirst.frc.team2473.framework.Devices;
+import org.usfirst.frc.team2473.robot.Robot;
+import org.usfirst.frc.team2473.robot.RobotMap;
+import org.usfirst.frc.team2473.robot.subsystems.DrivePower;
+
+
+public class PointTurn extends Command {
+	
+	private static double SLOW_POWER = 0.1;
+	private DrivePower slowDrivePower;
+	
+	private DrivePower power;
+	
+	private double prevDegrees;
+	private double degreesGoal;
+	
+	public PointTurn(double degrees, double power) {
```

Comment:
So we are using degrees then? (Honestly doesn't matter, the client probably understands degrees better)

Should we make this an int? We may not do == as we did in the drive for a certain distance class since the angle is a threshold.

ALWAYS REMEMBER TO DOCUMENT variables
What happens if degrees > ±360 ?